apps/dependencies/dependency-check-10-12-2020.csv

```diff
@@ -0,0 +1,195 @@
+"Project","ScanDate","DependencyName","DependencyPath","Description","License","Md5","Sha1","Identifiers","CPE","CVE","CWE","Vulnerability","Source","CVSSv2_Severity","CVSSv2_Score","CVSSv2","CVSSv3_BaseSeverity","CVSSv3_BaseScore","CVSSv3","CPE Confidence","Evidence Count"
```

Comment:
Looks like this file was committed on accident.

Reply:
Yes, will remove it. Thanks.